mkosi/distributions/gentoo.py

```diff
@@ -329,50 +309,24 @@ def merge_user_pkgs(self) -> None:
         if self.state.config.packages:
             self.invoke_emerge(pkgs=self.state.config.packages)
 
+    def cleanup(self) -> None:
```

Comment:
Please do this in clean_package_manager_metadata() instead of here

Reply:
not the same thing, image includes `portage`, those are simply sources, binary pkgs, etc. should i remove it for now? we can revisit this when we deal with image size.

Reply:
Sounds good, let's remove this for now